openvpn/middlewares/server/auth/middleware.go

```diff
@@ -176,8 +191,13 @@ func (m *middleware) authenticateClient(clientID, clientKey int, username, passw
 	}
 
 	if authenticated {
+		if err := filterSubnets(m.commandWriter, clientID, m.filter); err != nil {
```

Comment:
Lets not couple firewall configuration to authorisation hook.
Also would be good to have it in `node` repo as it is for Linux firewall rules.

Reply:
It's not a Linux firewall configuration. 
It's OpenVPN packet filtering which is applied during authorization.

Reply:
Extracted it into dedicated middleware, it seems like it works this way.